app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -334,7 +394,8 @@ impl component::Frp<Model> for Frp {
 
         // === Colors ===
         let colors = Colors::from_main_color(network, style, &input.set_color, &input.set_dimmed);
-        let params = entry::Params { colors: colors.clone_ref() };
+        let params =
+            entry::Params { colors: colors.clone_ref(), selection_layer: default() };
```

Comment:
We prefer 2 expressions with 2 variables than one multiline expression. Please, refactor `colors` to a separate variable, and make it one-liner.

Reply:
Done